css/template-1.css

```diff
@@ -0,0 +1,83 @@
+.full {
+  width: 100%;
+  padding-top: 170px;
+  padding-bottom: 170px;
+  background-color: #1e1e1e;
+}
+
+.garis {
+  background-color: #eaba3a;
+  width: 40px;
+  height: 5px;
+  position: absolute;
+  transform: translate(-50%, -10px);
+}
+
+
+/* List Ruang Publik */
+.list-aside li {
+  transform: translateX(-35px);
+  border-bottom: 1px solid  #808080;
+  padding: 10px;
+}
+
+.list-aside li a {
+  font-size: small;
+  text-decoration: none;
+  color: #808080;
+  transition: all 100ms;
+  margin-left: 15px;
+}
+
+.list-aside li span {
+  color: #808080;
+  transition: all 100ms;
+  transform: translateX(20px);
+}
+
+.list-aside li:hover > span,
+.list-aside li:hover > a{
+    color: #eaba3a;
+}
+
+/* Popular Post */
+.img-berita {
+    height: 100%;
+    object-fit: cover;
+}
+
+.popular-post div .tanggal {
+    font-size: small;
+}
+
+.article-berita div h5 a,
+.popular-post div .judul {
+    text-decoration: none;
+    color: #1e1e1e;
+    transition: all 100ms;
+}
+
+.article-berita div h5 a:hover,
+.popular-post div .judul:hover {
+    color: #eaba3a;
+}
+
+/* article berita */
+.tanggal-group {
+    max-height: 90px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5431765)